src/tui.rs

```diff
@@ -908,13 +960,15 @@ ctrl+z:
 ";
 
 pub enum InputAction {
-	Redraw,
-	JumpingToPage(usize),
-	Search(String),
-	QuitApp,
-	Invert,
-	Fullscreen,
-	SwitchRenderZoom(crate::FitOrFill)
+    Redraw,
```

Comment:
Weird that these got reformatted to use spaces... did you `cargo fmt` this? (if not, you should)